aws-billing/aws-billing.json

```diff
@@ -41,431 +54,347 @@
   },
   "description": "Visualize estimated AWS charges per AWS resource (EC2, S3, ...)",
   "editable": false,
+  "fiscalYearStartMonth": 0,
   "gnetId": 139,
   "graphTooltip": 0,
   "id": null,
-  "iteration": 1614364548685,
   "links": [],
   "panels": [
     {
-      "aliasColors": {},
-      "bars": false,
-      "dashLength": 10,
-      "dashes": false,
-      "datasource": "$datasource",
-      "editable": true,
-      "error": false,
+      "datasource": {
```

Comment:
This should be also updated, because input shouldn't be used. Use just `"uid": ${datasource}` everywhere instead

Reply:
yeah, ive adjusted it :)